router/core/src/main/scala/io/buoyant/router/Router.scala

```diff
@@ -302,6 +281,23 @@ object StackRouter {
       val default: PerClientParams = PerClientParams(Seq.empty)
     }
 
+    case class PathParams(prefix: PathMatcher, mk: Map[String, String] => Stack.Params)
```

Comment:
what's this Map[String, String] all about?  perhaps a typedef would make this clearer?